modules/installation-aws-supported-machine-types.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// installing/installing_aws/installing-aws-china.adoc
+// installing/installing_aws/installing-aws-customizations.adoc
+// installing/installing_aws/installing-aws-government-region.adoc
+// installing/installing_aws/installing-aws-network-customizations.adoc
+// installing/installing_aws/installing-aws-private.adoc
+// installing/installing_aws/installing-aws-user-infra.adoc
+// installing/installing_aws/installing-aws-vpc.adoc
+// installing/installing_aws/installing-restricted-networks-aws.adoc
+
+ifeval::["{context}" == "installing-aws-localzone"]
+:localzone:
+endif::[]
+
```

Comment:
Looking at this module closer, I think it is better served as a procedure `Identifying your AWS Outposts instance types` (purely a suggestion on the heading) Please remember to include all of the required metadata for a procedure assembly. The idea here is that you actually have to complete this procedure before you create the installation configuration file because you will have to specify the instance type for worker (compute) nodes when modifying install-config.yaml. I suggest placing this procedure module directly above Creating the installation configuration file.

Reply:
Renamed the module 'Identifying your AWS Outposts instance types' and placed it directly above 'Creating the installation configuration file.'